Update changelog for v`10.2.0`

ryanml opened the pull request:
## [10.2.0]
### Added
- [#12066](https://github.com/MetaMask/metamask-extension/pull/12066): Enable token conversion rates for primary currencies on some non-Mainnet networks
- [#12110](https://github.com/MetaMask/metamask-extension/pull/12110): Enable search tab on "Import Tokens" page on supported custom networks
  - Tab will be hidden when the `Use Token Detection` setting is disabled
- [#11798](https://github.com/MetaMask/metamask-extension/pull/11798): Add warning about fake tokens, and rename `Add Token` page to `Import Tokens`
- [#11755](https://github.com/MetaMask/metamask-extension/pull/11755): Add `Refresh list` button to the main page
  - This button is shown only when the `Use Token Detection` setting is disabled
- [#11851](https://github.com/MetaMask/metamask-extension/pull/11851): Add Token Detection toggle to experimental settings
  - This feature is experimental and is off by default, when enabled, it will add auto-detected tokens to a user's asset list
- [#12042](https://github.com/MetaMask/metamask-extension/pull/12042): Add Max Fee Per Gas to transaction breakdown
- [#11999](https://github.com/MetaMask/metamask-extension/pull/11999): Add eth_feeHistory to API callable by dapps
- [#11849](https://github.com/MetaMask/metamask-extension/pull/11849): Add subtitles for additional languages (Secret Recovery Phrase Video)
- [#11772](https://github.com/MetaMask/metamask-extension/pull/11772): Add tooltip to better explain possible effects of setting gas fees below MetaMask's estimates
- [#11796](https://github.com/MetaMask/metamask-extension/pull/11796): Add "Max fee" label to maximum fee amount in edit gas display

### Changed
- [#12019](https://github.com/MetaMask/metamask-extension/pull/12019): Show the user's address book name for a contract, instead of the contract address, when on the confirmation screen
- [#11915](https://github.com/MetaMask/metamask-extension/pull/11915): Improved time to see Swaps quotes
- [#11802](https://github.com/MetaMask/metamask-extension/pull/11802): Display the correct currency symbol in function type label for simple send transactions on custom networks
- [#11982](https://github.com/MetaMask/metamask-extension/pull/11982): Sort contacts alphabetically within each letter group in the contact list.
- [#12000](https://github.com/MetaMask/metamask-extension/pull/12000): Include the blocked URL in the link to report an incorrectly blocked page.
- [#11964](https://github.com/MetaMask/metamask-extension/pull/11964): Disable spell-check in the address input
- [#12013](https://github.com/MetaMask/metamask-extension/pull/12013): Improving warning text for eth_sign transactions
- [#11945](https://github.com/MetaMask/metamask-extension/pull/11945): Send user to account page upon adding a custom network
- [#11895](https://github.com/MetaMask/metamask-extension/pull/11895): Update header on Send page from `Add Recipient` -> `Send to`
- [#11850](https://github.com/MetaMask/metamask-extension/pull/11850): Use "Secret Recovery Phrase" text throughout the onboarding flow
- [#11795](https://github.com/MetaMask/metamask-extension/pull/11795): Autofocus the Amount field on the Send screen
- [#11889](https://github.com/MetaMask/metamask-extension/pull/11889): Improve gas controls descriptions
- [#11805](https://github.com/MetaMask/metamask-extension/pull/11805): Update tooltip text for gas fees to avoid referring to custom networks as "Ethereum".
- [#11845](https://github.com/MetaMask/metamask-extension/pull/11845): Wrapping and unwrapping native currencies (e.g. ETH -> WETH) through Swaps now uses the wrapping contracts directly
- [#12056](https://github.com/MetaMask/metamask-extension/pull/12056): Improve the text for the "Cancel Edit" button, and allow rejecting the transaction from the Edit screen.
- [#12098](https://github.com/MetaMask/metamask-extension/pull/12098): Scroll down to show all fields when "Advanced options" is toggled in the advanced gas options
- [#11944](https://github.com/MetaMask/metamask-extension/pull/11944): Improve the "Import account" page layout

### Fixed
- [#12140](https://github.com/MetaMask/metamask-extension/pull/12140): Prevent the user from entering more than 15 significant digits in numeric fields, such as Max Priority Fee
  - Previously, entering more than 15 significant digits would result in an error.
- [#12169](https://github.com/MetaMask/metamask-extension/pull/12169): Fix handling of erroneous ERC-20 Token amount on the confirmation screen
  - Previously, if you entered a token amount with too many decimal places, it would corrupt the token amount in the transaction. Now we truncate any decimals beyond what the token allows instead.
- [#12074](https://github.com/MetaMask/metamask-extension/pull/12074): Prevent the addition of a duplicate, zero-balance account after importing a wallet with seed phrase
- [#11967](https://github.com/MetaMask/metamask-extension/pull/11967): Fix the display of long contact names on the contact settings pages.
- [#12122](https://github.com/MetaMask/metamask-extension/pull/12122): Ensure failed speedups don't prevent further speedup attempts, and hide the Base Fee and Priority Fee fields when we don't have that information
- [#11963](https://github.com/MetaMask/metamask-extension/pull/11963): Show scrollbar in the accounts menu
  - This makes the accounts list scrollable for users with no mouse scroll wheel. 
- [#12058](https://github.com/MetaMask/metamask-extension/pull/12058): Fix clipping issue with long network names in the network dropdown
- [#12039](https://github.com/MetaMask/metamask-extension/pull/12039): Add missing padding at the bottom of the custom network form in the popup view
- [#11890](https://github.com/MetaMask/metamask-extension/pull/11890): Fix alignment of horizontal line shown under gas recommendations
- [#12244](https://github.com/MetaMask/metamask-extension/pull/12244): Fix form prefilling with values on the Build Quote page [Swaps]

ryanml committed:
[skip e2e] Update changelog for v10.2.0

adonesky1 commented on CHANGELOG.md:
```diff
@@ -7,228 +7,55 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-- EIP-1559 - Ensure transaction detail font-size and icon colors are consistent with Figma design ([#11735](https://github.com/MetaMask/metamask-extension/pull/11735))
-- EIP-1559 - Return null from GasTiming if on non-1559 network ([#11733](https://github.com/MetaMask/metamask-extension/pull/11733))
+### Added
+- [#11552](https://github.com/MetaMask/metamask-extension/pull/11552): Integrating new TokenRatesController to expand fiat conversion rates for some non-Mainnet networks
```
Sorry I know this was verbiage I gave you but just small nit change here:

```suggestion
- [#11552](https://github.com/MetaMask/metamask-extension/pull/11552): Integrating new TokenRatesController to enable fiat conversion rates for some non-Mainnet networks
```

ryanml replied:
done thanks!

tmashuang commented:
Fairly certain have all been a part of previous releases.

#11794: Add support for EIP-1559 to the transaction activity log
#11772: Add tooltip for negative gas timing
#11741: Add gasPrice to be used on non EIP-1559 networks for transaction details on confirm screen
#11751: Show advanced options, and hide radio buttons, for advanced gas settings users
#11889: Update gas controls copy
#11743: Show minimum native currency in banner when on testnets
#11805: Update tooltip text for gas fees

Gudahtt commented on CHANGELOG.md:
```diff
@@ -7,228 +7,53 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-- EIP-1559 - Ensure transaction detail font-size and icon colors are consistent with Figma design ([#11735](https://github.com/MetaMask/metamask-extension/pull/11735))
-- EIP-1559 - Return null from GasTiming if on non-1559 network ([#11733](https://github.com/MetaMask/metamask-extension/pull/11733))
+### Added
+- [#11552](https://github.com/MetaMask/metamask-extension/pull/11552): Integrating new TokenRatesController to expand fiat conversion rates for some non-Mainnet networks
```
The work described here was done in #12066 :thinking: #11552 was a prerequisite, but it wasn't a user-facing change.

Gudahtt commented on CHANGELOG.md:
```diff
@@ -7,228 +7,53 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#12058](https://github.com/MetaMask/metamask-extension/pull/12058): Fixed clipping issue with long network names (header)
+- [#12039](https://github.com/MetaMask/metamask-extension/pull/12039): Fixed padding issue at bottom of custom network form in popup view
+- [#11890](https://github.com/MetaMask/metamask-extension/pull/11890): Fixed horizontal line for gas recommendations
+- [#11787](https://github.com/MetaMask/metamask-extension/pull/11787): Fixed advanced gas form display issue when there are errors
```
This is missing from the v10 changelog for some reason though :thinking: It should be added there.

Gudahtt replied:
Huh. All of the ones I just suggested that you remove were omitted from the changelog.

ryanml replied:
Yeah looks like we need to go back and amend prior version entries

danjm commented on CHANGELOG.md:
```diff
@@ -7,228 +7,47 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-- EIP-1559 - Ensure transaction detail font-size and icon colors are consistent with Figma design ([#11735](https://github.com/MetaMask/metamask-extension/pull/11735))
-- EIP-1559 - Return null from GasTiming if on non-1559 network ([#11733](https://github.com/MetaMask/metamask-extension/pull/11733))
+### Added
+- [#12066](https://github.com/MetaMask/metamask-extension/pull/12066): Integrating new TokenRatesController to expand fiat conversion rates for some non-Mainnet networks
```
For a user facing description for 12066, perhaps "Enable fit conversion rates for primary currencies on some non-Mainnet networks"

Gudahtt replied:
I suppose you meant "Enable _fiat_ conversion rates"? It might be best to drop "fiat", as some of our primary currency choices aren't fiat.

ryanml replied:
Updated this to "Enable token conversion rates for primary currencies on some non-Mainnet networks"

danjm commented on CHANGELOG.md:
```diff
@@ -7,228 +7,47 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-- EIP-1559 - Return null from GasTiming if on non-1559 network ([#11733](https://github.com/MetaMask/metamask-extension/pull/11733))
+### Added
+- [#12066](https://github.com/MetaMask/metamask-extension/pull/12066): Integrating new TokenRatesController to expand fiat conversion rates for some non-Mainnet networks
+- [#12110](https://github.com/MetaMask/metamask-extension/pull/12110): Add search tab in import tokens for custom networks
```
For "Add search tab in import tokens for custom networks" and "Add `Refresh list`", these will be hidden if the toggle added with "Add Token Detection Toggle to Security & Privacy (Settings)" is turned off. Also, if that toggle is turned on, users will have more tokens detected auto-detected.

Someone we need to represent that the features hidden behind the toggle are experimental and toggled off by default, and that the toggle exists just to turn on those features.

Gudahtt replied:
This is one of the reasons I prefer the format of listing related PRs at the end of the change entry, so that we can explain related changes together, and add qualifiers as sub-bullet points.

If we're set on continuing to use this format, we could add a qualifier about the toggle under each change this applies to.

ryanml replied:
I think for now what @Gudahtt suggested about adding qualifiers makes the most sense until we change up the format of these entries, wdyt?

danjm replied:
@ryanml @Gudahtt that makes sense

ryanml replied:
Qualifiers added, let me know how that looks for those items

danjm commented on CHANGELOG.md:
```diff
@@ -7,228 +7,47 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#11798](https://github.com/MetaMask/metamask-extension/pull/11798): Add fake token warning and replacing `Add Token` labels with `Import Tokens`
+- [#11755](https://github.com/MetaMask/metamask-extension/pull/11755): Add `Refresh list` and `import tokens` link to the main page
+- [#11851](https://github.com/MetaMask/metamask-extension/pull/11851): Add Token Detection Toggle to Security & Privacy (Settings)
+- [#11845](https://github.com/MetaMask/metamask-extension/pull/11845): Add support for Swaps via a direct contract for wrapping and unwrapping
```
I might put 11845 under "Changed" and describe it as "Wrapping and unwrapping native currencies (e.g. ETH -> WETH)  through Swaps now uses the wrapping contracts directly"

ryanml replied:
updated

danjm commented on CHANGELOG.md:
```diff
@@ -7,228 +7,47 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#11851](https://github.com/MetaMask/metamask-extension/pull/11851): Add Token Detection Toggle to Security & Privacy (Settings)
+- [#11845](https://github.com/MetaMask/metamask-extension/pull/11845): Add support for Swaps via a direct contract for wrapping and unwrapping
+- [#12042](https://github.com/MetaMask/metamask-extension/pull/12042): Add Max Fee Per Gas to transaction breakdown
+- [#11999](https://github.com/MetaMask/metamask-extension/pull/11999): Add eth_feeHistory to safe method list
```
Maybe "Add eth_feeHistory to safe method list" should be "Add eth_feeHistory to API callable by dapps"

ryanml replied:
updated

danjm commented on CHANGELOG.md:
```diff
@@ -7,228 +7,47 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#12042](https://github.com/MetaMask/metamask-extension/pull/12042): Add Max Fee Per Gas to transaction breakdown
+- [#11999](https://github.com/MetaMask/metamask-extension/pull/11999): Add eth_feeHistory to safe method list
+- [#11849](https://github.com/MetaMask/metamask-extension/pull/11849): Add subtitles for additional languages (Secret Recovery Phrase Video)
+- [#11772](https://github.com/MetaMask/metamask-extension/pull/11772): Add tooltip for negative gas timing
```
Maybe change "Add tooltip for negative gas timing" to "Add tooltip to better explain possible effects of setting gas fees below MetaMask's estimates"

ryanml replied:
updated

danjm commented on CHANGELOG.md:
```diff
@@ -7,228 +7,47 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#11772](https://github.com/MetaMask/metamask-extension/pull/11772): Add tooltip for negative gas timing
+
+### Changed
+- [#12019](https://github.com/MetaMask/metamask-extension/pull/12019): Show local nicknames on confirmation screen when invoking a contract method
```
I think "Show local nicknames on confirmation screen when invoking a contract method" should be "Show the user's address book name for a contract, instead of the contract address, when on the confirmation screen"

ryanml replied:
updated

danjm commented on CHANGELOG.md:
```diff
@@ -7,228 +7,47 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#12019](https://github.com/MetaMask/metamask-extension/pull/12019): Show local nicknames on confirmation screen when invoking a contract method
+- [#11915](https://github.com/MetaMask/metamask-extension/pull/11915): Improved time to see Swaps quotes
+- [#11802](https://github.com/MetaMask/metamask-extension/pull/11802): Replace hardcoded sent ether label on confirmation screen
+- [#11982](https://github.com/MetaMask/metamask-extension/pull/11982): Sort contacts alphabetically
```
"Sort contacts alphabetically" -> "Sort addresses within letter groups in address book / contacts alphabetically"

ryanml replied:
updated

danjm commented on CHANGELOG.md:
```diff
@@ -7,228 +7,47 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#12074](https://github.com/MetaMask/metamask-extension/pull/12074): Fixed the addition of a duplicate, zero-balance account after importing a wallet with seed phrase
+- [#11944](https://github.com/MetaMask/metamask-extension/pull/11944): Fixed import account page layout
+- [#11967](https://github.com/MetaMask/metamask-extension/pull/11967): Fixed contact settings layout issues
+- [#12122](https://github.com/MetaMask/metamask-extension/pull/12122): Fixed issue with stuck speedup transactions
```
"Fixed issue with stuck speedup transactions" -> "Fixed stuck speedup transactions caused by prior failed speed up attempts"

ryanml replied:
updated

ryanml commented on CHANGELOG.md:
```diff
@@ -7,228 +7,47 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+### Fixed
+- [#12140](https://github.com/MetaMask/metamask-extension/pull/12140): Fixed `BigNumber` error in the max priority fee field
+- [#12169](https://github.com/MetaMask/metamask-extension/pull/12169): Fixed erroneous ERC-20 Token amount on the confirmation screen
+- [#12056](https://github.com/MetaMask/metamask-extension/pull/12056): Fixed duplicate cancel buttons on send screen when in editing stage
```
updated

ryanml commented on CHANGELOG.md:
```diff
@@ -7,228 +7,47 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#11796](https://github.com/MetaMask/metamask-extension/pull/11796): Update Max Fee label in the edit gas display
+
+### Fixed
+- [#12140](https://github.com/MetaMask/metamask-extension/pull/12140): Fixed `BigNumber` error in the max priority fee field
```
updated

ryanml commented on CHANGELOG.md:
```diff
@@ -7,228 +7,49 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#11798](https://github.com/MetaMask/metamask-extension/pull/11798): Add fake token warning and replacing `Add Token` labels with `Import Tokens`
+- [#11755](https://github.com/MetaMask/metamask-extension/pull/11755): Add `Refresh list` and `import tokens` link to the main page
+  - Links will be hidden when the `Use Token Detection` setting is disabled
+- [#11851](https://github.com/MetaMask/metamask-extension/pull/11851): Add Token Detection Toggle to Security & Privacy (Settings)
```
I think per @danjm's comment we'll need a qualifier under this too, any verbiage suggestions? cc: @Gudahtt

Gudahtt replied:
Maybe something like "This feature is experimental and is off by default".

ryanml replied:
How about: `This feature is experimental and is off by default, when enabled, it will add auto-detected tokens to a user's asset list`?

ryanml replied:
that or some other description of what it does

Gudahtt replied:
Sure, that sounds fine

ryanml replied:
done

Gudahtt commented:
It would be appreciated if your updates were left as separate commits by the way, rather than squashing them into the one commit. It makes it difficult to tell what has changed when they are squashed.

ryanml commented:
@Gudahtt - sorry, force of habit, I added a sub bullet under #12110 and #11755 in my last push

ryanml committed:
[skip e2e] adding qualifier to PR #11851

danjm reviewed (approved):
LGTM

Gudahtt commented on CHANGELOG.md:
```diff
@@ -7,228 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-- EIP-1559 - Return null from GasTiming if on non-1559 network ([#11733](https://github.com/MetaMask/metamask-extension/pull/11733))
+### Added
+- [#12066](https://github.com/MetaMask/metamask-extension/pull/12066): Enable token conversion rates for primary currencies on some non-Mainnet networks
+- [#12110](https://github.com/MetaMask/metamask-extension/pull/12110): Add search tab in import tokens for custom networks
```
Suggested re-wording:
```suggestion
- [#12110](https://github.com/MetaMask/metamask-extension/pull/12110): Enable search tab on "Import Tokens" page on supported custom networks
```

Gudahtt commented on CHANGELOG.md:
```diff
@@ -7,228 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#12066](https://github.com/MetaMask/metamask-extension/pull/12066): Enable token conversion rates for primary currencies on some non-Mainnet networks
+- [#12110](https://github.com/MetaMask/metamask-extension/pull/12110): Add search tab in import tokens for custom networks
+  - Tab will be hidden when the `Use Token Detection` setting is disabled
+- [#11798](https://github.com/MetaMask/metamask-extension/pull/11798): Add fake token warning and replacing `Add Token` labels with `Import Tokens`
```
The user-facing change here is to rename the page. Suggested re-wording:
```suggestion
- [#11798](https://github.com/MetaMask/metamask-extension/pull/11798): Add warning about fake tokens, and rename `Add Token` page to `Import Tokens`
```

Gudahtt commented on CHANGELOG.md:
```diff
@@ -7,228 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#12110](https://github.com/MetaMask/metamask-extension/pull/12110): Add search tab in import tokens for custom networks
+  - Tab will be hidden when the `Use Token Detection` setting is disabled
+- [#11798](https://github.com/MetaMask/metamask-extension/pull/11798): Add fake token warning and replacing `Add Token` labels with `Import Tokens`
+- [#11755](https://github.com/MetaMask/metamask-extension/pull/11755): Add `Refresh list` and `import tokens` link to the main page
+  - Links will be hidden when the `Use Token Detection` setting is disabled
```
The change from "Add Tokens" to "Import Tokens" is kinda implied by the #11798 entry, even if the change happened here :thinking: I think we can omit that part for the sake of simplifying this description. Right now this is incorrect - it wasn't _added_ here, we just changed the copy. That makes the qualifier incorrect as well ("Import tokens" is always shown).

This would straighten both issues out though:
```suggestion
- [#11755](https://github.com/MetaMask/metamask-extension/pull/11755): Add `Refresh list` button to the main page
  - This button is shown only when the `Use Token Detection` setting is disabled
```

Gudahtt commented on CHANGELOG.md:
```diff
@@ -7,228 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#11798](https://github.com/MetaMask/metamask-extension/pull/11798): Add fake token warning and replacing `Add Token` labels with `Import Tokens`
+- [#11755](https://github.com/MetaMask/metamask-extension/pull/11755): Add `Refresh list` and `import tokens` link to the main page
+  - Links will be hidden when the `Use Token Detection` setting is disabled
+- [#11851](https://github.com/MetaMask/metamask-extension/pull/11851): Add Token Detection Toggle to Security & Privacy (Settings)
```
This was moved to a new "Experimental" tab in #12136.

```suggestion
- [#11851](https://github.com/MetaMask/metamask-extension/pull/11851): Add Token Detection toggle to experimental settings
```

Gudahtt commented on CHANGELOG.md:
```diff
@@ -7,228 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+### Changed
+- [#12019](https://github.com/MetaMask/metamask-extension/pull/12019): Show the user's address book name for a contract, instead of the contract address, when on the confirmation screen
+- [#11915](https://github.com/MetaMask/metamask-extension/pull/11915): Improved time to see Swaps quotes
+- [#11802](https://github.com/MetaMask/metamask-extension/pull/11802): Replace hardcoded sent ether label on confirmation screen
```
This seems more like a bug fix to me.  Also I'd re-word it to make it clearer what changed.

Maybe something like this:
```suggestion
- [#11802](https://github.com/MetaMask/metamask-extension/pull/11802): Display the correct currency symbol in function type label for simple send transactions on custom networks
```

Seems a bit wordy :thinking: I struggled with this one.

Gudahtt commented on CHANGELOG.md:
```diff
@@ -7,228 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#12019](https://github.com/MetaMask/metamask-extension/pull/12019): Show the user's address book name for a contract, instead of the contract address, when on the confirmation screen
+- [#11915](https://github.com/MetaMask/metamask-extension/pull/11915): Improved time to see Swaps quotes
+- [#11802](https://github.com/MetaMask/metamask-extension/pull/11802): Replace hardcoded sent ether label on confirmation screen
+- [#11982](https://github.com/MetaMask/metamask-extension/pull/11982): Sort addresses within letter groups in address book / contacts alphabetically
```
We don't use the term "address book" anywhere in the UI. It's just called "Contacts". Also, it's contacts that we're sorting here, not addresses.

```suggestion
- [#11982](https://github.com/MetaMask/metamask-extension/pull/11982): Sort contacts alphabetically in contact list.
```
or
```suggestion
- [#11982](https://github.com/MetaMask/metamask-extension/pull/11982): Sort contacts alphabetically within each letter group in the contact list.
```

ryanml replied:
went with the second suggestion here

Gudahtt commented on CHANGELOG.md:
```diff
@@ -7,228 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#11915](https://github.com/MetaMask/metamask-extension/pull/11915): Improved time to see Swaps quotes
+- [#11802](https://github.com/MetaMask/metamask-extension/pull/11802): Replace hardcoded sent ether label on confirmation screen
+- [#11982](https://github.com/MetaMask/metamask-extension/pull/11982): Sort addresses within letter groups in address book / contacts alphabetically
+- [#12000](https://github.com/MetaMask/metamask-extension/pull/12000): Autofilling new issue link with url and domain (Phishing detect page)
```
The tense was wrong here, and I found the wording a bit confusing

```suggestion
- [#12000](https://github.com/MetaMask/metamask-extension/pull/12000): Include the blocked URL in the link to report an incorrectly blocked page.
```

Gudahtt commented on CHANGELOG.md:
```diff
@@ -7,228 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#11802](https://github.com/MetaMask/metamask-extension/pull/11802): Replace hardcoded sent ether label on confirmation screen
+- [#11982](https://github.com/MetaMask/metamask-extension/pull/11982): Sort addresses within letter groups in address book / contacts alphabetically
+- [#12000](https://github.com/MetaMask/metamask-extension/pull/12000): Autofilling new issue link with url and domain (Phishing detect page)
+- [#11964](https://github.com/MetaMask/metamask-extension/pull/11964): Disabled spell-check on ens input
```
Probably more understandable if we call it the address input. It's called the ENS input in the code, but in the UI we use it whenever we want to accept an address.

```suggestion
- [#11964](https://github.com/MetaMask/metamask-extension/pull/11964): Disable spell-check in the address input
```

Gudahtt commented on CHANGELOG.md:
```diff
@@ -7,228 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#11945](https://github.com/MetaMask/metamask-extension/pull/11945): Send user to account page upon adding a custom network
+- [#11895](https://github.com/MetaMask/metamask-extension/pull/11895): Update header on Send page from `Add Recipient` -> `Send to`
+- [#11850](https://github.com/MetaMask/metamask-extension/pull/11850): Use "Secret Recovery Phrase" text throughout the onboarding flow
+- [#11795](https://github.com/MetaMask/metamask-extension/pull/11795): Autofocus the unit field in send screen
```
This is called the "amount" field in the UI.
```suggestion
- [#11795](https://github.com/MetaMask/metamask-extension/pull/11795): Autofocus the Amount field on the Send screen
```

Gudahtt commented on CHANGELOG.md:
```diff
@@ -7,228 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#11895](https://github.com/MetaMask/metamask-extension/pull/11895): Update header on Send page from `Add Recipient` -> `Send to`
+- [#11850](https://github.com/MetaMask/metamask-extension/pull/11850): Use "Secret Recovery Phrase" text throughout the onboarding flow
+- [#11795](https://github.com/MetaMask/metamask-extension/pull/11795): Autofocus the unit field in send screen
+- [#11889](https://github.com/MetaMask/metamask-extension/pull/11889): Update gas controls copy
```
"copy" isn't a word that I'd expect users to understand, and it seems ambiguous.
```suggestion
- [#11889](https://github.com/MetaMask/metamask-extension/pull/11889): Improve gas controls descriptions
```

Gudahtt commented on CHANGELOG.md:
```diff
@@ -7,228 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#11850](https://github.com/MetaMask/metamask-extension/pull/11850): Use "Secret Recovery Phrase" text throughout the onboarding flow
+- [#11795](https://github.com/MetaMask/metamask-extension/pull/11795): Autofocus the unit field in send screen
+- [#11889](https://github.com/MetaMask/metamask-extension/pull/11889): Update gas controls copy
+- [#11805](https://github.com/MetaMask/metamask-extension/pull/11805): Update tooltip text for gas fees
```
We could describe this better I think
```suggestion
- [#11805](https://github.com/MetaMask/metamask-extension/pull/11805): Update tooltip text for gas fees to avoid referring to custom networks as "Ethereum".
```

Gudahtt commented on CHANGELOG.md:
```diff
@@ -7,228 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#11795](https://github.com/MetaMask/metamask-extension/pull/11795): Autofocus the unit field in send screen
+- [#11889](https://github.com/MetaMask/metamask-extension/pull/11889): Update gas controls copy
+- [#11805](https://github.com/MetaMask/metamask-extension/pull/11805): Update tooltip text for gas fees
+- [#11796](https://github.com/MetaMask/metamask-extension/pull/11796): Update Max Fee label in the edit gas display
```
It's unclear to me from reading that PR what has changed about this label :thinking:

ryanml replied:
I've made an effort to make this clearer

Gudahtt replied:
Thanks, that sounds OK to me!

Gudahtt commented on CHANGELOG.md:
```diff
@@ -7,228 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+
+### Fixed
+- [#12140](https://github.com/MetaMask/metamask-extension/pull/12140): Fixed max priority fee field so that there are not errors when attempting to enter more than 15 significant digits
+- [#12169](https://github.com/MetaMask/metamask-extension/pull/12169): Fixed erroneous ERC-20 Token amount on the confirmation screen
```
```suggestion
- [#12169](https://github.com/MetaMask/metamask-extension/pull/12169): Fix handling of erroneous ERC-20 Token amount on the confirmation screen
  - Previously, if you entered a token amount with too many decimal places, it would corrupt the token amount in the transaction. Now we truncate any decimals beyond what the token allows instead.
```

Gudahtt commented on CHANGELOG.md:
```diff
@@ -7,228 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+### Fixed
+- [#12140](https://github.com/MetaMask/metamask-extension/pull/12140): Fixed max priority fee field so that there are not errors when attempting to enter more than 15 significant digits
+- [#12169](https://github.com/MetaMask/metamask-extension/pull/12169): Fixed erroneous ERC-20 Token amount on the confirmation screen
+- [#12074](https://github.com/MetaMask/metamask-extension/pull/12074): Fixed the addition of a duplicate, zero-balance account after importing a wallet with seed phrase
```
```suggestion
- [#12074](https://github.com/MetaMask/metamask-extension/pull/12074): Prevent the addition of a duplicate, zero-balance account after importing a wallet with seed phrase
```

Gudahtt commented on CHANGELOG.md:
```diff
@@ -7,228 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#12140](https://github.com/MetaMask/metamask-extension/pull/12140): Fixed max priority fee field so that there are not errors when attempting to enter more than 15 significant digits
+- [#12169](https://github.com/MetaMask/metamask-extension/pull/12169): Fixed erroneous ERC-20 Token amount on the confirmation screen
+- [#12074](https://github.com/MetaMask/metamask-extension/pull/12074): Fixed the addition of a duplicate, zero-balance account after importing a wallet with seed phrase
+- [#11944](https://github.com/MetaMask/metamask-extension/pull/11944): Fixed import account page layout
```
This does not seem like a bug fix - this is a design improvement.
```suggestion
- [#11944](https://github.com/MetaMask/metamask-extension/pull/11944): Improve the "Import account" page layout
```

ryanml replied:
updated

Gudahtt commented on CHANGELOG.md:
```diff
@@ -7,228 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#12169](https://github.com/MetaMask/metamask-extension/pull/12169): Fixed erroneous ERC-20 Token amount on the confirmation screen
+- [#12074](https://github.com/MetaMask/metamask-extension/pull/12074): Fixed the addition of a duplicate, zero-balance account after importing a wallet with seed phrase
+- [#11944](https://github.com/MetaMask/metamask-extension/pull/11944): Fixed import account page layout
+- [#11967](https://github.com/MetaMask/metamask-extension/pull/11967): Fixed contact settings layout issues
```
My impression is that the thing fixed here is primarily the display of long contact names. This might be a better description:
```suggestion
- [#11967](https://github.com/MetaMask/metamask-extension/pull/11967): Fix the display of long contact names on the contact settings pages.
```

Gudahtt commented on CHANGELOG.md:
```diff
@@ -7,228 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#12074](https://github.com/MetaMask/metamask-extension/pull/12074): Fixed the addition of a duplicate, zero-balance account after importing a wallet with seed phrase
+- [#11944](https://github.com/MetaMask/metamask-extension/pull/11944): Fixed import account page layout
+- [#11967](https://github.com/MetaMask/metamask-extension/pull/11967): Fixed contact settings layout issues
+- [#12122](https://github.com/MetaMask/metamask-extension/pull/12122): Fixed stuck speedup transactions caused by prior failed speed up attempts
```
Suggested wording improvement:
```suggestion
- [#12122](https://github.com/MetaMask/metamask-extension/pull/12122): Ensure failed speedups don't prevent further speedup attempts, and hide the Base Fee and Priority Fee fields when we don't have that information
```

Gudahtt commented on CHANGELOG.md:
```diff
@@ -7,228 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#11944](https://github.com/MetaMask/metamask-extension/pull/11944): Fixed import account page layout
+- [#11967](https://github.com/MetaMask/metamask-extension/pull/11967): Fixed contact settings layout issues
+- [#12122](https://github.com/MetaMask/metamask-extension/pull/12122): Fixed stuck speedup transactions caused by prior failed speed up attempts
+- [#12098](https://github.com/MetaMask/metamask-extension/pull/12098): Fixed scrolling input issue in advanced gas options
```
This seems more like a UX improvement :thinking: 

I don't think it's correct to describe this as fixing a scrolling issue. This PR _adds_ scrolling. Suggested rewording:
```suggestion
- [#12098](https://github.com/MetaMask/metamask-extension/pull/12098): Scroll down to show all fields when "Advanced options" is toggled in the advanced gas options
```

ryanml replied:
updated

Gudahtt commented on CHANGELOG.md:
```diff
@@ -7,228 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#11967](https://github.com/MetaMask/metamask-extension/pull/11967): Fixed contact settings layout issues
+- [#12122](https://github.com/MetaMask/metamask-extension/pull/12122): Fixed stuck speedup transactions caused by prior failed speed up attempts
+- [#12098](https://github.com/MetaMask/metamask-extension/pull/12098): Fixed scrolling input issue in advanced gas options
+- [#11963](https://github.com/MetaMask/metamask-extension/pull/11963): Fixed scrolling issue in accounts dropdown
```
```suggestion
- [#11963](https://github.com/MetaMask/metamask-extension/pull/11963): Show scrollbar in the accounts menu
  - This makes the accounts list scrollable for users with no mouse scroll wheel. 
```

Gudahtt commented on CHANGELOG.md:
```diff
@@ -7,228 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#12122](https://github.com/MetaMask/metamask-extension/pull/12122): Fixed stuck speedup transactions caused by prior failed speed up attempts
+- [#12098](https://github.com/MetaMask/metamask-extension/pull/12098): Fixed scrolling input issue in advanced gas options
+- [#11963](https://github.com/MetaMask/metamask-extension/pull/11963): Fixed scrolling issue in accounts dropdown
+- [#12058](https://github.com/MetaMask/metamask-extension/pull/12058): Fixed clipping issue with long network names (header)
```
Nit: The tense is wrong here, and I think calling this the network dropdown would be clearer than "(header)".
```suggestion
- [#12058](https://github.com/MetaMask/metamask-extension/pull/12058): Fix clipping issue with long network names in the network dropdown
```

Gudahtt commented on CHANGELOG.md:
```diff
@@ -7,228 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#12098](https://github.com/MetaMask/metamask-extension/pull/12098): Fixed scrolling input issue in advanced gas options
+- [#11963](https://github.com/MetaMask/metamask-extension/pull/11963): Fixed scrolling issue in accounts dropdown
+- [#12058](https://github.com/MetaMask/metamask-extension/pull/12058): Fixed clipping issue with long network names (header)
+- [#12039](https://github.com/MetaMask/metamask-extension/pull/12039): Fixed padding issue at bottom of custom network form in popup view
```
Updated to clarify what the padding issue was
```suggestion
- [#12039](https://github.com/MetaMask/metamask-extension/pull/12039): Add missing padding at the bottom of the custom network form in the popup view
```

Gudahtt commented on CHANGELOG.md:
```diff
@@ -7,228 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#11963](https://github.com/MetaMask/metamask-extension/pull/11963): Fixed scrolling issue in accounts dropdown
+- [#12058](https://github.com/MetaMask/metamask-extension/pull/12058): Fixed clipping issue with long network names (header)
+- [#12039](https://github.com/MetaMask/metamask-extension/pull/12039): Fixed padding issue at bottom of custom network form in popup view
+- [#11890](https://github.com/MetaMask/metamask-extension/pull/11890): Fixed horizontal line for gas recommendations
```
Updated to explain what was fixed
```suggestion
- [#11890](https://github.com/MetaMask/metamask-extension/pull/11890): Fix alignment of horizontal line shown under gas recommendations
```

Gudahtt commented:
I suspect that https://github.com/MetaMask/metamask-extension/pull/12244 belongs in this list as well (assuming that bug was found on prod, not on `develop`. Not 100% sure what that situation was).

ryanml committed:
Updating logline text per review comments

Update 12110

Co-authored-by: Mark Stacey <markjstacey@gmail.com>

Update 11798

Co-authored-by: Mark Stacey <markjstacey@gmail.com>

Update 11851

Co-authored-by: Mark Stacey <markjstacey@gmail.com>

Update 11755

Co-authored-by: Mark Stacey <markjstacey@gmail.com>

Update 11802

Co-authored-by: Mark Stacey <markjstacey@gmail.com>

Update 11795

Co-authored-by: Mark Stacey <markjstacey@gmail.com>

Update 11889

Co-authored-by: Mark Stacey <markjstacey@gmail.com>

Update 11805

Co-authored-by: Mark Stacey <markjstacey@gmail.com>

Update 11964

Co-authored-by: Mark Stacey <markjstacey@gmail.com>

Update 11890

Co-authored-by: Mark Stacey <markjstacey@gmail.com>

Update 12039

Co-authored-by: Mark Stacey <markjstacey@gmail.com>

Update 12058

Co-authored-by: Mark Stacey <markjstacey@gmail.com>

Update 11963

Co-authored-by: Mark Stacey <markjstacey@gmail.com>

Update 12056

Co-authored-by: Mark Stacey <markjstacey@gmail.com>

Update 12169

Co-authored-by: Mark Stacey <markjstacey@gmail.com>

Update 12122

Co-authored-by: Mark Stacey <markjstacey@gmail.com>

Update 11967

Co-authored-by: Mark Stacey <markjstacey@gmail.com>

Update 11982

Co-authored-by: Mark Stacey <markjstacey@gmail.com>

Update 12074

Co-authored-by: Mark Stacey <markjstacey@gmail.com>

Update 12000

Co-authored-by: Mark Stacey <markjstacey@gmail.com>

ryanml commented on CHANGELOG.md:
```diff
@@ -7,228 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#11805](https://github.com/MetaMask/metamask-extension/pull/11805): Update tooltip text for gas fees
+- [#11796](https://github.com/MetaMask/metamask-extension/pull/11796): Update Max Fee label in the edit gas display
+- [#11845](https://github.com/MetaMask/metamask-extension/pull/11845): Wrapping and unwrapping native currencies (e.g. ETH -> WETH) through Swaps now uses the wrapping contracts directly
+- [#12056](https://github.com/MetaMask/metamask-extension/pull/12056): Updated labels and behaviour of duplicate cancel buttons on the send screen when editing a transaction
```
lgtm

ryanml committed:
[skip e2e] Update log entries for #12098, #11944, #12140, and #11796

ryanml commented on CHANGELOG.md:
```diff
@@ -7,228 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#12056](https://github.com/MetaMask/metamask-extension/pull/12056): Updated labels and behaviour of duplicate cancel buttons on the send screen when editing a transaction
+
+### Fixed
+- [#12140](https://github.com/MetaMask/metamask-extension/pull/12140): Fixed max priority fee field so that there are not errors when attempting to enter more than 15 significant digits
```
updated

Gudahtt commented:
All of the wording changes seem good! Though I had suggested moving 12098 and 11944 from "Fixed" to "Changed" - any thoughts on those? Or do you think they're better where they are?

And we still need to add #12244

ryanml committed:
[skip e2e] Move #12098, #11944, from "Fixed" -> "Changed", add #12244

ryanml commented:
@Gudahtt - agreed, those two have been moved to the "Changed" section. Additionally #12244 has been added

Gudahtt reviewed (approved):
LGTM!
